### PR TITLE
Assignment rewriting and integer casting

### PR DESCRIPTION
**Pull request by bknaepen**
- An alternative derivation of the rocket assignment equations is proposed (already discussed privately).
- An integer conversion has been added in the notebook 03_02_convectionSchemes.ipynb to avoid a repetitive warning.

### CODE REVIEW

File: lessons/01_phugoid/Rocket_Assignment.ipynb [hunk 1]
```diff
@@ -90,109 +90,38 @@
    "source": [
     "In case you are kind of confused about the rocket equations, here we show how to get to them. \n",
     "\n",
-    "We will start from the widely used [Reynolds' transport theorem](http://en.wikipedia.org/wiki/Reynolds_transport_theorem). For an extensive property $N$ in a volume $\\Omega$, with corresponding intensive property $\\eta = N/m$, the Reynolds' transport theorem reads\n",
+    "Newton's second law states that the acceleration of our vehicule times it mass is equal to all the forces acting on it. Therefore,\n",
```
Comment (gforsyth) [inline lessons/01_phugoid/Rocket_Assignment.ipynb]: I would change this to 

"Newton's second law states that the acceleration of the vehicle times its mass is equal to all the forces acting on it."

File: lessons/01_phugoid/Rocket_Assignment.ipynb [hunk 2]
```diff
@@ -90,109 +90,38 @@
...
     "\n",
-    "In practical terms, we're only interested in the $y$ component. Vectors $\\mathbf{g}$ and $\\mathbf{v}_e$ are pointing down, which gives the scalar equation:\n",
+    "\\begin{align}\n",
+    "{\\bf F}_{gas\\rightarrow rocket} = -{\\bf F}_{rocket\\rightarrow gas} = -\\dot m_p {\\bf v}_e\n",
+    "\\end{align}\n",
     "\n",
-    "$$\n",
-    "-(m_c + m_p) g - \\frac{1}{2} \\rho_a v^2 A  C_D - (m_c + m_p)\\frac{{\\rm d} v}{{\\rm d}t} =  -v_e \\dot{m}_p.\n",
-    "$$\n",
+    "If we collect all the forces acting on the rocket we finaly have:\n",
```
Comment (gforsyth) [inline lessons/01_phugoid/Rocket_Assignment.ipynb]: *finally

File: lessons/01_phugoid/Rocket_Assignment.ipynb [hunk 3]
```diff
@@ -90,109 +90,38 @@
...
-    "$$\n",
+    "\\begin{align}\n",
+    "{\\bf F}_g&= (m_s+m_p)\\bf{g} \\quad (\\rm{Gravity}),\\\\\n",
+    "{\\bf F}_d&= - \\frac{1}{2} \\rho_a \\mathbf{v} |\\mathbf{v}| A  C_D \\quad (\\rm{Drag}).\n",
+    "\\end{align}\n",
     "\n",
-    "where $\\mathbf{v}_e$ is the exhaust velocity of the burnt propellant coming out of the nozzle with respect to the control volume, $\\rho_p$ is the density of the propellant, and $\\dot{m}_p$ is the mass flow rate of propellant. By saying that $\\rho = \\rho_p$ we are considering that the mass of air is negligible compared to the mass of propellant coming out of the nozzle.\n",
+    "We also need to consider the force resulting from the ejection of the propelant. During an interval $dt$, the engine of the rocket ejects downwards a mass of propelant given by $\\dot m_p dt$. Relative to the rocket, the speed of the ejected burning gas is assumed constant and equal to $v_e$ (the exhaust speed). The momentum variation induced on the exhaust gas by the engine during that interval is therefore, $d{\\bf p}_{gas} = \\dot m_p {\\bf v}_e dt$. Again using Newton's second law we conclude that the force applied by the rocket on the gas is,\n",
```
Comment (gforsyth) [inline lessons/01_phugoid/Rocket_Assignment.ipynb]: *propellant

**Comment by bknaepen** [comment]
Above corrections made.

**Pull request merged**